qiskit/algorithms/evolvers/evolution_result.py

```diff
@@ -27,14 +29,18 @@ def __init__(
         self,
         evolved_state: Union[StateFn, QuantumCircuit, OperatorBase],
         aux_ops_evaluated: Optional[ListOrDict[Tuple[complex, complex]]] = None,
+        observables: Optional[ListOrDict[Tuple[np.ndarray, np.ndarray]]] = None,
     ):
         """
         Args:
             evolved_state: An evolved quantum state.
             aux_ops_evaluated: Optional list of observables for which expected values on an evolved
                 state are calculated. These values are in fact tuples formatted as (mean, standard
                 deviation).
+            observables: Optional list of observables for which expected values for each timestep.
+             These values are in fact tuples formatted as (mean, standard deviation).
         """
 
         self.evolved_state = evolved_state
         self.aux_ops_evaluated = aux_ops_evaluated
+        self.observables = observables
```

Comment:
I guess I missed this when the evolution result was done but we normally have private fields and setter/getter properties to document these as an end user does not use the constructor normally rather they will access the fields. We could do the Attributes here too so as to have them documented properly